exchanges/bybit/bybit_test.go

```diff
@@ -3121,6 +3108,158 @@ func TestPushData(t *testing.T) {
 	}
 }
 
+func TestWsTicker(t *testing.T) {
```

Comment:
I am unable to have this run successfully, neither in mock or live tests:
```
    bybit_test.go:3120: 
        	Error Trace:	/Users//go/src/github.com/thrasher-corp/gocryptotrader/internal/testing/exchange/exchange.go:172
        	            				/Users//go/src/github.com/thrasher-corp/gocryptotrader/exchanges/bybit/bybit_test.go:3120
        	Error:      	Received unexpected error:
        	            	pair not found for btc28jun24-60000-p options
        	Test:       	TestWsTicker
        	Messages:   	Fixture message should not error:
        	            	{"topic":"tickers.BTC-28JUN24-60000-P","ts":1715742949283,"type":"snapshot","id":"tickers.BTC-28JUN24-60000-P-6222585724-1715742949283","data":{"symbol":"BTC-28JUN24-60000-P","bidPrice":"3475","bidSize":"10.14","bidIv":"0.5479","askPrice":"3520","askSize":"2.5","askIv":"0.5534","lastPrice":"3565","highPrice24h":"3715","lowPrice24h":"3555","markPrice":"3502.0715721","indexPrice":"61912.8","markPriceIv":"0.5513","underlyingPrice":"62588.50709846","openInterest":"29.35","turnover24h":"101005.6875725","volume24h":"1.62","totalVolume":"45","totalTurnover":"2911557","delta":"-0.37596534","gamma":"0.00003161","vega":"82.65324199","theta":"-51.54651685","predictedDeliveryPrice":"0","change24h":"0.02148998"}}
    bybit_test.go:3126: 
        	Error Trace:	/Users//go/src/github.com/thrasher-corp/gocryptotrader/exchanges/bybit/bybit_test.go:3126
        	Error:      	"0x14001ed6cc0" should have 8 item(s), but has 7
        	Test:       	TestWsTicker
        	Messages:   	Should see correct number of tickers
```

Reply:
Yeah, that got broken after 24th.
Simples really. Don't update pairs.
Susceptible to breakage if someone regenerates the pairs in testconfig for bybit, but they'll know cos it'll break 🤷 

Fixed 22179fa